aptos-move/framework/aptos-framework/doc/fungible_asset.md

```diff
@@ -1504,7 +1506,7 @@ Get the symbol of the fungible asset from the <code>metadata</code> object.
 
 ## Function `decimals`
 
-Get the decimals from the <code>metadata</code> object.
+Get the decimals of the fungible asset from the <code>metadata</code> object.
```

Comment:
I made this change only because all the other metadata field getter comment strings had this and this one didn't